projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad-checklist.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Launchpad Checklist API endpoint
+ *
+ * @package automattic/jetpack-mu-wpcom
+ * @since 1.1.0
+ */
+
+/**
+ * Fetches Launchpad checklist data for the site
+ *
+ * @since 1.1.0
+ */
+class WPCOM_REST_API_V2_Endpoint_Launchpad_Checklist extends WP_REST_Controller {
+
+	/**
+	 * Class constructor
+	 */
+	public function __construct() {
+		require_once __DIR__ . '/../launchpad/launchpad.php';
+
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'launchpad/checklist';
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register our routes.
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_data' ),
+					'permission_callback' => array( $this, 'can_access' ),
```

Comment:
@agrullon95 When you get a chance, could you add an `args` entry here for `checklist_slug` with sanitization/validation as needed, similar to https://github.com/Automattic/jetpack/pull/29841/files#r1163346776?

Reply:
https://github.com/Automattic/jetpack/pull/30112